lang/en/docs/package-json.md

```diff
@@ -358,7 +358,9 @@ Bundled dependencies are an array of package names that will be bundled together
 }
 ```
 
-If your package only allows one version of a given dependency, and you'd like to enforce the same behavior as [`yarn install --flat`]({{url_base}}/docs/cli/install#toc-yarn-install-flat) on the command line, set this to `true`.
+If your package only allows one version of a given dependency, and you'd like to enforce the same behavior as [`yarn install --flat`] on the command line, set this to `true`.
+
+  [`yarn install --flat`]: {{url_base}}/docs/cli/install#toc-yarn-install-flat
```

Comment:
sorry, I just saw this, not the other change! Could you use inline links like in the rest of the documentation?

Reply:
No, because it is reused later. I mean, I can duplicate it, but that wan't whoever made the initial non-working link meant to do and neither is it what I like to see...

Reply:
I didn't see that it's reused, sorry, it's fine to duplicate & just fix the other link then :)